project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
> I think we wrote cross posts :-)

Yes, weird to reply a comment and find it comes after new one. Anyway thanks for helping :)